tool/travis.sh

```diff
@@ -10,17 +10,60 @@ set -e
 # Print out the Dart version in use.
 dart --version
 
-# Add globally activated packages to the path.
-export PATH="$PATH":"~/.pub-cache/bin"
+if [ "$USE_FLUTTER_SDK" = true ] ; then
```

Comment:
There is duplicated code between the USE_FLUTTER_SDK and else branch.
Clean it up before landing by running the tests that don't require flutter first.

Reply:
Done.